### DISCUSSION TITLE
USD for 3ds Max is now open source

### DISCUSSION THREAD

**Discussion by wallworm** [Autodesk/3dsmax-usd#7] [Announcements]
It’s with great excitement that the USD team announces that USD for 3ds Max is now open source. This announcement is a reinforcement of our commitment to open-source standards and our continuing quest to maintain and grow trust in the community.

USD for 3ds Max is only the latest in a broad swathe of open-source projects around USD here at Autodesk. Among the others you will find [USD for Maya](https://github.com/Autodesk/maya-usd), [Arnold USD](https://github.com/Autodesk/arnold-usd), [Maya Hydra](https://github.com/Autodesk/maya-hydra) and [Bifrost USD](https://github.com/Autodesk/bifrost-usd). All these projects have a common goal of bringing our tools together more cohesively while exposing the power and versatility of OpenUSD.

What can you expect from the USD for 3ds Max project? For the near term, we will continue to work on the project internally and provide regular updates on this repository. This initial release is the code for USD for 3ds Max 0.9 that was released in October 2024.

We hope that this announcement is as exciting to you as it is to us. Feel free to share with us at any time ideas or issues you encounter. We are committed to helping you succeed at bringing your ideas to life with our tools—and we are devoted to making production in USD more and more accessible.

To learn more about the [features in USD for 3ds Max, see the online documentation](https://help.autodesk.com/view/3DSMAX/2025/ENU/?guid=GUID-04F1DF51-0079-4DF8-8457-5AD12B6C0673).

**Comment by ryanspletzer** [comment]
[heart]         Ryan Spletzer reacted to your message:
________________________________
From: Shawn Olson ***@***.***>
Sent: Wednesday, December 4, 2024 4:45:00 PM
To: Autodesk/3dsmax-usd ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Autodesk/3dsmax-usd] USD for 3ds Max is now open source (Discussion #7)

EXTERNAL EMAIL : Do not click any links or open any attachments unless you trust the sender and know the content is safe.


It’s with great excitement that the USD team announces that USD for 3ds Max is now open source. This announcement is a reinforcement of our commitment to Open Source standards and our continuing quest to maintain and grow trust in the community.

USD for 3ds Max is only the latest in a broad swathe of open-source projects around USD here at Autodesk. Among the others you will find USD for Maya<https://github.com/Autodesk/maya-usd>, Arnold USD<https://github.com/Autodesk/arnold-usd>, Maya Hydra<https://github.com/Autodesk/maya-hydra> and Bifrost USD<https://github.com/Autodesk/bifrost-usd>. All these projects have a common goal of bringing our tools together more cohesively while exposing the power and versatility of OpenUSD.

What can you expect from the USD for 3ds Max project? For the near term, we will continue to work on the project internally and provide regular on this repository. This initial release is the code for USD for 3ds Max 0.9 that was released in October 2024.

We hope that this announcement is as exciting to you as it is to us. Feel free to share with us at any time ideas or issues you encounter. We are committed to helping you succeed at bringing your ideas to life with our tools—and we are devoted to making production in USD more and more accessible.

To learn more about the features in USD for 3ds Max, see the online documentation<https://help.autodesk.com/view/3DSMAX/2025/ENU/?guid=GUID-04F1DF51-0079-4DF8-8457-5AD12B6C0673>.
